ngraph/frontend/paddlepaddle/src/op/assign_value.cpp

```diff
@@ -0,0 +1,62 @@
+//*****************************************************************************
```

Comment:
Change to 3-line copyrights. We have confirmed that correct copyright notice shall be
// Copyright (C) 2018-2021 Intel Corporation
// SPDX-License-Identifier: Apache-2.0
//